### ISSUE TITLE
<feature request> Parsec disconnect

### ISSUE THREAD

**Issue by zulc22** [jarrettgilliam/AutoGame#18]
could a command be ran on a _disconnect_ from Parsec? my use case is for making a couple of scripts to disable all but my primary monitor whenever I connect and re-enable them on disconnect. this could also be used to run `taskkill /im [launcher.exe]` to close the game launcher on disconnect.

**Comment by jarrettgilliam** [comment]
Good idea! It seems feasible. I could see this being useful for a lot of things really.

As far as closing the game launcher on disconnect, I think we could add first class support for that like we have for starting it.

I sort of envision being able to trigger multiple actions from a single event as well. Really just a much more flexible system.

That being said, it would be sort of a large work effort. I'll think on it.
